distribution/test/itests/test-itests-common/src/main/java/org/codice/ddf/itests/common/opensearch/OpenSearchSourceProperties.java

```diff
@@ -0,0 +1,37 @@
+/**
+ * Copyright (c) Codice Foundation
+ * <p>
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ * <p>
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package org.codice.ddf.itests.common.opensearch;
+
+import java.util.HashMap;
+
+import org.codice.ddf.itests.common.AbstractIntegrationTest;
+
+public class OpenSearchSourceProperties extends HashMap<String, Object> {
```

Comment:
❗️Instead of extending from `HashMap`, could we make this class a little more like a stand-alone one that wraps the `OpenSearchSource` feature? For instance, this class could have a `createManagedService` method that takes in a username/password and uses the `ServiceManager` to start a new instance of the service. That method could also take care of waiting until the feature is actually up and running before returning.

Doing this would also allow us to add open search source related operation and validation methods to this class if needed and make it more re-usable down the road.